assets/training/model_management/src/azureml/model/mgmt/processors/common/vision_utils.py

```diff
@@ -23,6 +25,21 @@
 # Uncomment the following line for mlflow debug mode
 # logging.getLogger("mlflow").setLevel(logging.DEBUG)
 
+def save_image(output_folder: str, img: PIL.Image.Image, format: str) -> str:
```

Comment:
Where do we use this save_image method?

Reply:
In convert to mlflow for text to image model we have  used save_image  trowed an error cannot import name 'save_image' from 'vision_utils' [[compvis_stable_diffusion_v1_4 - Azure AI | Machine Learning Studio](https://ml.azure.com/experiments/id/af133222-4f28-4f36-92cf-1dfcb7cad70f/runs/great_bulb_vvrfmd1gw7?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-suvrat&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)](url)